doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1294,6 +1400,48 @@ Serial LTE Modem
 
 The issues in this section are related to the :ref:`serial_lte_modem` application.
 
+.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0
+
+NCSDK-19278: Modem traces captured through UART are corrupted if RTT logs are simultaneously captured.
+  When capturing modem traces through UART with `Cellular Monitor`_ app and simultaneously capturing RTT logs, for example, with J-Link RTT Viewer, the modem trace misses packets, and captured packets might have incorrect information.
+
+  **Affected platforms:** nRF9160, nRF9161
+
+  **Workaround:** If you need to capture modem traces and RTT logs at the same time, add the following change to :file:`nrf9160dk_nrf9160_ns.overlay` or :file:`nrf9161dk_nrf9161_ns.overlay`, depending on the board you are using.
+  Otherwise, you can choose not to capture RTT logs.
+  Having RTT logs enabled does not cause this issue.
+
+  .. parsed-literal::
+     :class: highlight
+
+     &uart1 {
+      hw-flow-control;
+     };
+
+  This increases the overall power consumption even when Serial LTE Modem is in sleep mode.
+
+.. rst-class:: v2-5-0
+
+NCSDK-19278: UART hardware flow control is not working for some scenarios with Thingy:91
+  After we run out of UART receive buffers and hardware flow control is enabled, we can sometimes still receive ``UART_RX_STOPPED`` with overflow status and lose data that is sent to Thingy:91.
+  This depends on timing and what has been transmitted previously.
+
+  **Affected platforms:** Thingy:91
+
+  **Workaround:** Use the :kconfig:option:`CONFIG_SLM_UART_RX_BUF_SIZE` Kconfig option to ensure that there is adequate buffer space for the worst-case scenario.
+  Additionally, you may disable hw-flow-control from :file:`boards\thingy91_nrf9160_ns.overlay`.
```

Comment:
```suggestion
.. rst-class:: v2-5-0

NCSDK-19278: SLM attempts to use UART hardware flow control even though Connectivity Bridge does not support it
  With Thingy:91, Connectivity Bridge in nRF52840 terminates the USB traffic and sends the traffic via UART to SLM in nRF9160. Connectivity Bridge does not enable UART hardware flow control and since both ends need to enable it, SLM should not enable it either.
  Without hardware flow control, the buffer sizes must be set according to worst-case scenario. In this case, SLM uses the default buffer size of 3 x 256 bytes and will drop incoming UART traffic after the buffers are full.

  **Affected platforms:** Thingy:91

  **Workaround:** Set the :kconfig:option:`CONFIG_SLM_UART_RX_BUF_SIZE` Kconfig option to 2048 to ensure that there is adequate buffer space to receive traffic from the Connectivity Bridge and disable the hw-flow-control from :file:`boards\thingy91_nrf9160_ns.overlay`.
  If even larger buffer spaces are required, the :kconfig:option:`CONFIG_BRIDGE_BUF_SIZE` Kconfig option for Connectivity Bridge and the :kconfig:option:`CONFIG_SLM_UART_RX_BUF_SIZE` Kconfig option for SLM, should be set accordingly.
```